src/quo2/components/list_items/account_list_card/view.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo2.components.list-items.account-list-card.view
+  (:require
+    [quo2.components.avatars.account-avatar.view :as account-avatar]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.list-items.account-list-card.style :as style]))
+
+(defn- internal-view
+  [{:keys [theme account-props networks state action on-press]}]
+  [rn/view
+   {:style               (style/container state theme)
+    :accessibility-label :container}
+   [rn/view
+    {:style {:flex-direction :row
+             :align-items    :center}}
+    [account-avatar/view account-props]
+    [rn/view {:style {:margin-left 8}}
+     [text/text
+      {:weight :semi-bold
+       :size   :paragraph-2} (:name account-props)]
+     [text/text {:size :paragraph-2}
+      (map (fn [network]
+             ^{:key (str network)}
+             [text/text
+              {:size  :paragraph-2
+               :style {:color (get colors/networks network)}} (str (subs (name network) 0 3) ":")])
```

Comment:
Is this really correct for the user? I think the logic to take the first 3 characters can be flawed. It looks nice for `eth` if we are abbreviating from `ethereum`, but not for other networks.

Should we at least lowercase the string? That way, something like `zkSync` would become `zks` (still weird IMO).

Reply:
Perhaps this info is coming from status go side too? Maybe best to check if there's already something there and then we don't need to add formatting on the front end

Reply:
> Perhaps this info is coming from status go side too? Maybe best to check if there's already something there and then we don't need to add formatting on the front end

👍🏼 Makes sense too. There's the argument in favor of not coupling status-go with the presentation layer. To me, status-go should only ever return stable and complete network names and leave the formatting to clients.

Reply:
Good point @ilmotta. We still don't what exactly comes from status-go as we still have not started with the flows. There were other components were I had similar thoughts about what comes from status-go. I think UI-wise it is good for now and we can update later on if needed.